docs/content/middlewares/http/corazawaf.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Traefik Coraza WAF Documentation"
+description: "The HTTP coraza middleware in Traefik Proxy implements web application firewall capability to Traefik. Read the technical documentation."
+---
+
+# Coraza WAF
+
+The HTTP [Coraza](https://coraza.io/) middleware in Traefik Proxy implements web application firewall capability to Traefik.
+To get help writing rules https://coraza.io/docs/tutorials/introduction/ and https://coreruleset.org/docs/ are excellent places to start. 
+
+## Configuration Examples
+
+```yaml tab="Kubernetes"
+# Denying /admin path
+apiVersion: traefik.io/v1alpha1
+kind: Middleware
+metadata:
+  name: waf
+spec:
+  corazaWAF:
+    directives: |
+      SecRuleEngine On
+      SecRule REQUEST_URI "@streq /admin" "id:101,phase:1,t:lowercase,log,deny"
+```
+
+## Configuration Options
+
+### `directives`
+
+The `directives` string contains configurable waf rules for the middleware.
+
+As an example we might want to block /admin path in production.
+
+```yaml tab="Kubernetes"
+apiVersion: traefik.io/v1alpha1
+kind: Middleware
+metadata:
+  name: waf
+spec:
+  corazaWAF:
+    directives: |
+      SecRuleEngine On
+      SecRule REQUEST_URI "@streq /admin" "id:101,phase:1,t:lowercase,log,deny"
+```
+
+### `crsEnabled`
```

Comment:
For the sake of consistency I'd strongly recommend call it `loadOWASPCRS` as in https://github.com/corazawaf/coraza-caddy/blob/main/coraza.go#L34C31-L34C45